tigerpath/majors_and_certificates/majors/PHI_2018.json

```diff
@@ -15,143 +15,508 @@
+              "course_list": [
+                "PHI *",
+                "POL *",
+                "HIS *",
+              ]
+            }
           ]
         },
-        {
-          "name": "History of Philosophy",
-          "max_counted": 1,
-          "min_needed": 2,
-          "explanation": "",
-          "course_list": [
-            "PHI 200",
-            "PHI 205",
-            "PHI 300",
-            "PHI 301",
-            "PHI 302",
-            "PHI 303",
-            "PHI 304",
-            "PHI 306",
-            "PHI 332",
-            "PHI 333",
-            "PHI 335",
-            "PHI 338"
-          ]
-        }
-      ]
-    },
-    {
-      "name": "Interdisciplinary Options",
-      "max_counted": null,
-      "min_needed": null,
-      "explanation": "",
-      "req_list": [
         {
           "name": "Political Philosophy",
           "max_counted": 1,
           "min_needed": "ALL",
-          "explanation": "Political Philosophy. Senior concentrators doing their theses in political philosophy have the option of substituting for the usual distribution requirement (two courses in each of three areas plus two unrestricted courses) the following: two courses from among those listed under the Department of Politics as courses in political theory; two philosophy courses in the ethics and philosophy of value area; two philosophy courses in one other philosophy distribution area; and two philosophy courses unrestricted as to distribution area.",
-          "no_req": true
+          "explanation": "Senior concentrators doing their theses in political philosophy have the option of substituting for the usual distribution requirement (two courses in each of three areas plus two unrestricted courses) the following: two courses from among those listed under the Department of Politics as courses in political theory; two philosophy courses in the ethics and philosophy of value area; two philosophy courses in one other philosophy distribution area; and two philosophy courses unrestricted as to distribution area.",
+          "req_list": [
+            {
+              "name": "Political Theory",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "<i>Editor's Note: Some of these courses fall under this requirement but TigerPath does not verify this.</i>\n\nTwo courses from among those listed under the Department of Politics as courses in political theory.",
+              "course_list": [
+                "POL *"
+              ]
+            },
+            {
+              "name": "Ethics and Philosophy of Value",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Two philosophy courses in the ethics and philosophy of value area.",
+              "course_list": [

```

Comment:
This looks like another major where factoring out the course lists could be useful. Just noting it. No need to do anything right now.